build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -689,9 +692,6 @@ message DirectoryNode {
   // represented. See [Digest][build.bazel.remote.execution.v2.Digest]
   // for information about how to take the digest of a proto message.
   Digest digest = 2;
-
-  // The node properties of the DirectoryNode.
-  repeated NodeProperty node_properties = 3;
```

Comment:
I think these properties may also be removed from one or two other places (e.g., output directories).

Reply:
Right, thanks!

Reply:
I understand why properties should be added to the directory message but why are these being removed?

Reply:
Otherwise all directories except the top-level one would have two `node_properties` lists, and we should only need one per directory.

Reply:
ah, i see